app/src/test/java/org/simple/clinic/newentry/PatientEntryScreenControllerTest.kt

```diff
@@ -475,10 +475,12 @@ class PatientEntryScreenControllerTest {
     verify(patientRegisteredCount).set(any())
   }
 
-  @Test // TODO: Migrate to Mobius
+  @Test
```

Comment:
Since the tests for Mobius will co-exist with older tests, what do you think about adding a comment to indicate/segregate both the tests? 
Or to have Mobius tests in a separate file?
Else it might be hard to just read the tests and figure out how much has been migrated or how to write future tests in Mobius.

Reply:
@sanchita Noted, Mobius tests will live inside their own classes. We are touching (please take this with a grain of salt) any of the tests for now.